src/api/routes/edition.js

```diff
@@ -223,10 +250,82 @@ router.get('/:bbid/identifiers',
  */
 
 router.get('/:bbid/relationships',
-	makeEntityLoader('Edition', relationshipsRelations, editionError),
+	makeEntityLoader('Edition', utils.relationshipsRelations, editionError),
 	async (req, res, next) => {
 		const editionRelationshipList = await getEntityRelationships(res.locals.entity);
 		return res.status(200).send(editionRelationshipList);
 	});
 
+/**
+ *	@swagger
+ * '/edition':
+ *   get:
+ *     tags:
+ *       - Browse Requests
+ *     summary: Get list of Edition which are related to the other entity
+ *     description: Returns the list of Edition, When one of the bbid of Work or Author or Publisher or EditionGroup is passed as query parameter
+ *     operationId: getRelatedEditionByBbid
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: author
+ *         in: query
+ *         description: BBID of the Author
+ *         required: false
+ *         type: bbid
+ *       - name: work
+ *         in: query
+ *         description: BBID of the Work
+ *         required: false
+ *         type: bbid
+ *       - name: publisher
+ *         in: query
+ *         description: BBID of the Publisher
+ *         required: false
+ *         type: bbid
+ *       - name: edition-group
+ *         in: query
+ *         description: BBID of the EditionGroup
+ *         required: false
+ *         type: bbid
+ *     responses:
+ *       200:
+ *         description: List of Editions which are related any of one other entity
+ *         schema:
+ *             $ref: '#/definitions/BrowsedEditions'
+ *       404:
+ *         description: Related entity not found
+ *       406:
+ *         description: Invalid BBID paased in query params
+ */
+
+router.get('/',
+	validateBrowseRequestQueryParameters(['author', 'edition', 'edition-group', 'work', 'publisher']),
+	makeEntityLoader(null, utils.relationshipsRelations, 'Entity not found', true),
```

Comment:
@prabalsingh24 The comment above is about loading related entities that do not appear in relationships.

There are two cases where this happens, both for Edition entities: Edition Group and Publisher.
Both can be defined as relationships (Edition has Edition Group… and Edition was published by…), but I am fixing these and replacing them respectively with the editionGroupBbid and publisherSetId columns on the edition_data table.
Don't ask me why they were modified like that, it was before my time and I never got a good answer for it.

So for Edition<->Edition Group link the comment above was implemented (search for `if (req.query.modelType === 'EditionGroup') …` and `if (req.query.modelType === 'Edition') …`).

Now the same should be implemented for Edition<->Publisher links. An Edition entity can have a publisherSet with an array of Publisher entities in it (currently the UI only allows adding one. go figure…).
The Edition ORM model has a `publisherSet()` method and the Publisher ORM model has an `editions(fetchOptions)` method to simplify all this.

To illustrate the issue, here's an example of a publisher that appears in two Editions' publisherSet: https://bookbrainz.org/publisher/9dc3a2f2-3d7d-4dda-b25b-b6de0de898fd
Fetching  http://localhost:9098/1/edition?publisher=9dc3a2f2-3d7d-4dda-b25b-b6de0de898fd returns no results, since they are not defined as relationships.